apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -143,8 +143,18 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
             const tempProfileInfo: Map<string, string> = new Map<string, string>();
 
             profileSchema.forEach((schema: ProfileSchema) => {
+
+                // this splits for the sub-attributes
                 const schemaNames = schema.name.split(".");
```

Comment:
```suggestion
                const schemaNames: string[] = schema.name.split(".");
```

Reply:
Check all the variables as well.

Reply:
Fixed with 2fec35b068334696276f45f5b8045f98bd5c8151